MidasCivil_Adapter/Convert/ToMidasCivil/Properties/FromMetadata.cs

```diff
@@ -0,0 +1,92 @@
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Text;
+using BH.oM.Adapters.MidasCivil;
+using System.Collections.Generic;
+
+namespace BH.Adapter.Adapters.MidasCivil
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static StringBuilder FromMetadata(Metadata metadata)
+        {
+            StringBuilder midasMetadata = new StringBuilder();
+            List<Review> reviews = metadata.Reviews;
+            midasMetadata.AppendLine("*PROJINFO");
+            midasMetadata.AppendLine($"PROJECT={metadata.ProjectNumber}");
+            midasMetadata.AppendLine($";DESIGNSTAGE={metadata.DesignStage}");
+            midasMetadata.AppendLine($";PROJECTLEAD={metadata.ProjectLead}");
+            midasMetadata.AppendLine($"REVISION={metadata.Revision}");
+            midasMetadata.AppendLine($"USER={metadata.Author}");
+            midasMetadata.AppendLine($"EMAIL={metadata.Email}");
+            midasMetadata.AppendLine($"ADDRESS={metadata.Location.Replace(System.Environment.NewLine, ", ")}");
+            midasMetadata.AppendLine($"CLIENT={metadata.Client}");
+            midasMetadata.AppendLine($"TITLE={metadata.ProjectName}");
+            midasMetadata.AppendLine($"ENGINEER={metadata.Author}");
+            midasMetadata.AppendLine($"EDATE={metadata.CreationDate.ToString("yyyy-MM-dd")}");
+            midasMetadata.AppendLine($";DESCRIPTION={metadata.Description}");
+            midasMetadata.AppendLine($";DISCIPLINE={metadata.Discipline}");
+
+            if(reviews.Count > 4)
+            {
+                reviews.RemoveRange(1, reviews.Count - 2);
```

Comment:
```suggestion
```

We shouldn't modify the data - raising a warning for an incomplete push is adequate. 

You can then just append the lines for CHECK1, 2 and 3 using the appropriate reviews - I tried pushing 5 reviews but I only get two (see updated test file).

Then the following statements can be contained within an `else`.